rector.php

```diff
@@ -7,11 +7,11 @@
 
 return RectorConfig::configure()
     ->withPaths([
-        __DIR__.'/app',
-        __DIR__.'/bootstrap/app.php',
-        __DIR__.'/config',
-        __DIR__.'/database',
-        __DIR__.'/public',
+        __DIR__ . '/app',
+        __DIR__ . '/bootstrap/app.php',
+        __DIR__ . '/config',
+        __DIR__ . '/database',
+        __DIR__ . '/public',
```

Comment:
Keep changes relevant to the focus of the PR.

Reply:
Yeah, I think this was added by my editor's formatter. Let me know if it's fine now.